kubernetes/overlays/prod/ondemand-patch.yaml

```diff
@@ -157,7 +134,7 @@ spec:
         - name: GIT_REPO
           value: https://github.com/slaclab/slac-ood-desktop.git
         - name: GIT_RELEASE
-          value: s3df-dev
```

Comment:
Not sure why this was pointing at s3df-dev rather than s3df - assuming it was supposed to be pointing at s3df previously, it should now be pointing at prod.